x-pack/legacy/plugins/apm/typings/elasticsearch.ts

```diff
@@ -82,7 +88,11 @@ declare module 'elasticsearch' {
         max: AggregatedValue;
         min: AggregatedValue;
         sum: AggregatedValue;
-        terms: BucketAggregation<AggregationOption[AggregationName]>;
+        value_count: AggregatedValue;
+        terms: BucketAggregation<
+          AggregationOption[AggregationName],
+          string | number
```

Comment:
this is correct (TIL!) but it's causing some type errors downstream. Adding a `.toString()` should be enough in most of the cases.